src/v2/Apps/Artwork/Components/OtherWorks/index.tsx

```diff
@@ -145,6 +165,8 @@ export const OtherWorksFragmentContainer = createFragmentContainer<{
         }
       }
       ...RelatedWorksArtworkGrid_artwork
+      ...ArtistSeriesArtworkRail_artwork
```

Comment:
Since this winds up doing work (fetching artist series, and then artworks for the first one), let's leave the fragment out entirely unless the user has the lab feature. That's passed down from the routes file.